Jasmine/Common/Extensions/UIVIewController+Alert.swift

```diff
@@ -0,0 +1,33 @@
+import UIKit
+
+extension UIViewController {
+
+    /// Returns an alert controller with cancel button
+    ///
+    /// - Parameters:
+    ///   - title: title of the alert
+    ///   - message: message for the alert
+    /// - Returns: UIAlertController with cancel button
+    func getAlertControllerWithCancel(title: String, message: String? = nil) -> UIAlertController {
```

Comment:
It's common for a UIVC to present an alert controller though. Where would be a better place to put it?